skycoin/skywire
# Change build path of binaries to build folder

Did you run `make format && make check`?

Fixes #	

 Changes:	
- build path of binaries moved from `root` to `./build`
- added new makefile rules

How to test this PR:
1. Run `make build`
2. Run `cd ./build`
3. Run `./skywire-cli config gen -ir`
4. Run `./skywire-visor`
5. Check if it works on Mac, Linux and Windows
6. Check other related makefile rules
7. Test Autoconfig
8. Run `make build1` for old functionality i.e. build binaries in root
